.github/workflows/release.yaml

```diff
@@ -28,14 +34,12 @@ jobs:
           key: cache-${{ env.cache-version }}-go-${{ env.go-version }}-${{ hashFiles('v2/Makefile') }}
       - run: make setup
         if: steps.cache-tools.outputs.cache-hit != 'true'
-      - run: make image
       - name: Login to ghcr.io
         run: echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u $GITHUB_ACTOR --password-stdin
-      - name: Push to ghcr.io
+      - run: make build
```

Comment:
It appears that this step can be removed altogether now as there isn't any `build` target in Makefile.